generated/nidigital/nidigital_service.cpp

```diff
@@ -1734,6 +1964,116 @@ namespace nidigital_grpc {
     }
   }
 
+  //---------------------------------------------------------------------
+  //---------------------------------------------------------------------
+  ::grpc::Status NiDigitalService::GetPinResultsPinInformation(::grpc::ServerContext* context, const GetPinResultsPinInformationRequest* request, GetPinResultsPinInformationResponse* response)
```

Comment:
@Akshata440 this API has multiple array parameters whose size mechanism is `ivi_dance_with_a_twist`, although with same `value_twist`. This seems to have generated fine. Is that handled explicitly, or did it just happen to work? :)

If this could potentially break we should probably write unit test for this in Akshata's PR.

Reply:
Yeah it works because twist value is used for initialising the each of the output array parameters [here](https://github.com/doshirohan/grpc-device/blob/4eebbb6bbcc56bbd97c2165b43b0384ca94c7863/source/codegen/templates/service_helpers.mako#L254)

Reply:
Ah, I see. So no separate unit test is necessary then. Consider this comment resolved.